src/theia/sfm/feature_extractor_and_matcher.h

```diff
@@ -91,10 +91,13 @@ class FeatureExtractorAndMatcher {
   bool AddImage(const std::string& image_filepath,
                 const CameraIntrinsicsPrior& intrinsics);
 
-  // Add a mask to the 'image_masks_' map.
-  bool AddMaskForFeaturesExtraction(
-      const std::string& image_filepath,
-      const std::string& mask_filepath);
+  // Assignes a mask to an image.
+  // The white part of the mask indicates the area for the keypoints extraction.
+  // The mask is a basic black and white image (jpg, png, tif etc.) and its name
+  // must content the associated image's name (e.g. 'image0001_mask.jpg' is the
+  // mask of 'image0001.png').
+  bool AddMaskForFeaturesExtraction(const std::string& image_filepath,
```

Comment:
This comment is not correct. The limitation of the mask name only applies to the build_reconstructions.cc file. In this interface, we specify the connection between the image file and the mask file, so the mask does not need to be named any particular way as far as this class is concerned.

Also please specify that white is 1.0 and black is 0.0 in the comments.